examples/graphics/material-translucent-specular.html

```diff
@@ -0,0 +1,195 @@
+                    material0.normalMap = normalMap;
+                    material0.diffuse = new pc.Color(0.6, 0.6, 0.9);
+                    material0.diffuseTint = true;
+                    material0.metalness = 1.0;
+                    material0.shininess = 0.0;
+                    material0.bumpiness = 0.7;
+                    material0.useMetalness = true;
+
+                    material0.update();
+
+                    //createSphere(-0.5, 0, -1, material0,2);
+                    //createSphere( 0.5, 0, -1, material0,2);
+
+                    material = new pc.StandardMaterial();
+                    material.diffuseMap = diffuseMap;
+                    material.metalnessMap = otherMap;
+                    material.metalnessMapChannel = 'r';
+                    material.glossMap = otherMap;
+                    material.glossMapChannel = 'g';
+                    material.normalMap = normalMap;
+                    material.diffuse = new pc.Color(0.6, 0.6, 0.9);
+                    material.diffuseTint = true;
+                    material.metalness = 1.0;
+                    material.shininess = 90.0;
+                    material.bumpiness = 0.7;
+                    material.useMetalness = true;
+
+                    material.clearCoat = 0.25;
+                    material.clearCoatGlossiness = 0.9;
+
+                    material.blendType = pc.BLEND_NORMAL;
+                    material.opacity = 0.5;
+                    material.opacityFadesSpecRefl = true;
+                    material.alphaWrite = false;
+
+                    material.update();
+
+                    createSphere(-0.5, 0, 0, material);
+
+                    clearCoatMaterial = new pc.StandardMaterial();
+                    clearCoatMaterial.diffuseMap = diffuseMap;
+                    clearCoatMaterial.metalnessMap = otherMap;
+                    clearCoatMaterial.metalnessMapChannel = 'r';
+                    clearCoatMaterial.glossMap = otherMap;
+                    clearCoatMaterial.glossMapChannel = 'g';
+                    clearCoatMaterial.normalMap = normalMap;
+                    clearCoatMaterial.diffuse = new pc.Color(0.6, 0.6, 0.9);
+                    clearCoatMaterial.diffuseTint = true;
+                    clearCoatMaterial.metalness = 1.0;
+                    clearCoatMaterial.shininess = 90;
+                    clearCoatMaterial.bumpiness = 0.7;
+                    clearCoatMaterial.useMetalness = true;
+
+                    clearCoatMaterial.clearCoat = 0.25;
+                    clearCoatMaterial.clearCoatGlossiness = 0.9;
+
+                    clearCoatMaterial.blendType = pc.BLEND_PREMULTIPLIED;
+                    clearCoatMaterial.opacity = 0.5;
+                    clearCoatMaterial.opacityFadesSpecRefl = false;
+                    clearCoatMaterial.alphaWrite = false;
+
+                    clearCoatMaterial.update();
+
+                    createSphere(0.5, 0, 0, clearCoatMaterial);
```

Comment:
you could probably create a function for the shared material set up and only set up differences individually to make it cleaner

Reply:
I completely switched out the engine example - see picture in the description